easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'Jansson'
+version = "2.13.1"
+
+homepage = 'https://www.digip.org/jansson/'
+description = """Jansson is a C library for encoding, decoding and manipulating JSON data.
+ Its main features and design principles are:
+ * Simple and intuitive API and data model
+ * Comprehensive documentation
+ * No dependencies on other libraries
+ * Full Unicode support (UTF-8)
+ * Extensive test suite"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/akheron/jansson/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f22901582138e3203959c9257cf83eba9929ac41d7be4a42557213a22ebcc7a0']
+
+preconfigopts = 'autoreconf -i && '
```

Comment:
Since this calls preconfigopts, this needs a to add builddep on Autotools

Reply:
Added Autoconf, thanks for spotting that.

Reply:
Swapped Autoconf with Autotools as suggested.